### PR TITLE
PLG-212: Free trial - Invite user

### PR DESCRIPTION
**Pull request by LaurentPetard**
Rename the Free-Trial bundle to be able to add the main bundle in `EE/tria/`

### CODE REVIEW

File: src/Akeneo/Platform/Bundle/UIBundle/Resources/public/templates/menu/user-navigation.html [hunk 1]
```diff
@@ -1,3 +1,5 @@
+<div id="dropzone-before-user-navigation"></div>
```
Comment (olivierpontier) [inline src/Akeneo/Platform/Bundle/UIBundle/Resources/public/templates/menu/user-navigation.html:1]: We shoudl add a comment to explain the container is used in the Free Trial?
```suggestion
<div id="dropzone-before-user-navigation"><% /** Container used in Free Trial **/ %></div>
```

Or override the template directly in FT?

Comment (olivierpontier) [reply src/Akeneo/Platform/Bundle/UIBundle/Resources/public/templates/menu/user-navigation.html:1]: As we already override the `user-navigation` module in the FT we can also override the template to add this dedicated container directly in the FT

Comment (LaurentPetard) [reply src/Akeneo/Platform/Bundle/UIBundle/Resources/public/templates/menu/user-navigation.html:1]: I don't know what is the best solution. I let you choose

Comment (olivierpontier) [reply src/Akeneo/Platform/Bundle/UIBundle/Resources/public/templates/menu/user-navigation.html:1]: Could you move move this part in the FT, please?

**Review by olivierpontier — APPROVED** [review]
GTM after moving the override of the user navigation template in the FT

**Review by BryanS29-akeneo — APPROVED** [review]

**Review by rouliane — APPROVED** [review]

**Pull request merged**